hangzhounet/values.yaml

```
@@ -0,0 +1,149 @@
+accounts:
+  ecadlabs:
+    # unconfirmed
+    # tz1cjyja1TU6fiyiFav3mFAdnDsCReJ12hPD
+    key: edpkvP1NXoo8vhYbPSvXdy466EHoYWBpf6zmjghB2p3DwJPjbB5nsf
+    type: public
+    is_bootstrap_baker_account: true
+    bootstrap_balance: "5000000000000"
+  smartpy:
+    # unconfirmed
+    # tz1VpvtSaSxKvykrqajFJTZqCXgoVJ5cKaM1
+    key: edpktv8Advoagbpo5TiyyDnmC1Mqne6vJMaXQ2vvuzUfJnBfMKJmdC
+    type: public
+    is_bootstrap_baker_account: true
+    bootstrap_balance: "5000000000000"
+  bruno:
+    # unconfirmed
+    # tz1SqQebt7jSyf5pUKKBjx34kySkGr1hXqtw
+    key: edpkuu3jmS3kbaLYTmyVaPHaxowC1KP8fDSrLoJhcvip1Cm2tK5QCd
+    type: public
+    is_bootstrap_baker_account: true
+    bootstrap_balance: "5000000000000"
+  tezosil:
+    # unconfirmed
+    # tz1VWasoyFGAWZt5K2qZRzP3cWzv3z7MMhP8
+    key: edpkvQj8r3YigN6QFsDRaKQCoxDNhKK4whDUkFkw6Gom36d86uhEbv
+    type: public
+    is_bootstrap_baker_account: true
+    bootstrap_balance: "5000000000000"
+  pirbo:
+    # unconfirmed
+    # tz1NFs6yP2sXd5vAAbR43bbDRpV2nahDZope
+    key: edpkuGH2TWTQ5JrwzBLGRu5is1h7vcv5e5LN45VfqmXS5zaTUMpopH
+    type: public
+    is_bootstrap_baker_account: true
+    bootstrap_balance: "5000000000000"
+  marco:
+    # unconfirmed
+    # tz1T8UYSbVuRm6CdhjvwCfXsKXb4yL9ai9Q3
+    key: edpkvMWHXeETjH4i2h2ngkG2y6aVdgtW99pQ3SuHJoWNEPKANzVRLT
+    type: public
+    is_bootstrap_baker_account: false
+    bootstrap_balance: "5000000000000"
+  tz1THUNA:
+    # tz1THUNARo58aD5YdNGYPnWNnvd8yHPrMdsF 
+    # Romain's faucet
+    key: edpkuh3WSwziyd6nXqDXeh48vZHp5ETxFd43EdkbzhmW98BvuLXK81
+    type: public
+    is_bootstrap_baker_account: false
+    bootstrap_balance: "500000000000000"
+  oxheadbaker:
+    # Oxhead Alpha's baker hosted in teztnets.xyz infra
+    # tz1foXHgRzdYdaLgX6XhpZGxbBv42LZ6ubvE
+    bootstrap_balance: "5000000000000"
+    is_bootstrap_baker_account: true
+    type: secret
+    # key injected by pulumi
+  oxheadfree:
+    # an account with a lot of money in it
+    # tz1foxFdz2ieSj8P9YxKYBTEqYbgFXXEeiQY
+    key: edpkubSgCoMjB2a6vPWRNRHfGa6HUFfFyEUKoEQhWC8osnav8QzBdJ
+    bootstrap_balance: "5000000000000000"
+    is_bootstrap_baker_account: false
+    type: public
+
+node_config_network:
+  activation_account_name: oxheadbaker
+  chain_name: TEZOS_HANGZHOUNET_2021-09-22T15:00:00Z
+  genesis:
+    block: BLockGenesisGenesisGenesisGenesisGenesisd4299hBGVoU
```

```suggestion
    block: BLockGenesisGenesisGenesisGenesisGenesis94ee4bjQDg5
```
This is a fresh hash. It's better to have a different genesis block hash for each network if possible, to avoid mixing the chains by accident (even though the chain id already prevents that).